core/core-documentation.el

```diff
@@ -111,7 +147,7 @@
              :base-directory ,spacemacs-docs-directory
              :base-extension "org"
              :publishing-directory ,(concat publish-target "doc/")
-             :publishing-function org-html-publish-to-html
+             :publishing-function spacemacs//spacemacs-doc-publishing-function
```

Comment:
I think I can use `advice` instead. But I have to use `org-html-publish-to-htm` in two different places with the two different advices. 

This way the code is clearer.